eras/conway/impl/src/Cardano/Ledger/Conway/TxInfo.hs

```diff
@@ -497,9 +498,15 @@ transTxCert = \case
   UnRegTxCert stakeCred ->
     PV3.TxCertUnRegStaking (transCred stakeCred) Nothing
   RegDepositTxCert stakeCred deposit ->
-    PV3.TxCertRegStaking (transCred stakeCred) (Just (transCoinToLovelace deposit))
+    let transDeposit
```

Comment:
It would very useful to add a comment, or even better a haddock to `transTxCert` that describes why this bootstrap phase check is necessary.

Reply:
Yes, I totally forgot. Added now, thanks!